modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -112,16 +112,21 @@ This first phase is implicit. During this phase, the runtime performs all the "b
 Mule exposes the batch job instance ID through the `batchJobInstanceId` variable. This variable is available in every step and the on-complete phase.
 +
 . Mule creates a persistent queue and associates it with the new batch job instance.
++
+The queue name is prefixed with `BSQ`, and you can see it in Anypoint Runtime Manager:
++
+image:mruntime-batch-bsq.png[Batch step queue transaction log]
+
 . For each item generated by the splitter, Mule creates a record and stores it in the queue. This activity is "all or nothing" – Mule either successfully generates and queues a record for every item, or the whole message fails during this phase.
 . Mule presents the batch job instance, with all its queued-up records, to the first batch step for processing.
 
-After this phase completes, the flow will continue.
-
-The next phase, "Process," is asynchronous, meaning that the flow will not wait for the batch job to finish processing all records.
+After this phase completes, the flow continues its execution without waiting for the batch job to finish processing all records. This behavior occurs because the next phase, _Process_, is asynchronous.
 
 === Process
 
-During the "Process" phase, the runtime begins processing the records in the batch asynchronously. Each record moves through the processors in the first batch step, then is sent back to the original queue while it waits to be processed by the second batch step and so on until every record has passed through every batch step. Only one queue exists, and records are picked out of it for each batch step, processed, and then sent back to it; each record keeps track of what stages it has been processed through while it sits on this queue. Note that a batch job instance does not wait for all its queued records to finish processing in one batch step before pushing any of them to the next batch step. Queues are persistent.
+During the _Process_ phase, Mule begins processing the records in the batch asynchronously. Each record moves through the processors in the first batch step, then is sent back to the original queue while it waits to be processed by the second batch step and so on until every record has passed through every batch step. Only one queue exists per each batch job instance. The records are picked out of the queue for each batch step, they are processed, and then sent back to the queue; each record keeps track of what stages it has been processed through while it sits on this queue. However, a batch job instance might create additional queues for internal purposes, for example if a batch step contains an aggregator. 
```

Comment:
The text needs some commas, which I added. It is not clear what the "and so on" means. Can you remove it? "each record keeps track..." is a run-on sentence. I split it into two sentences. I made slight wording changes that read better or remove some unnecessary words.

During the _Process_ phase, Mule begins processing the records in the batch asynchronously. Each record moves through the processors in the first batch step, and is then sent back to the original queue while it waits to be processed by the second batch step, and so on, until every record has passed through every batch step. Only one queue exists per each batch job instance. The records are picked out of the queue for each batch step, processed, and then sent back to the queue. Each record keeps track of the stages it was processed through while it sits on this queue. However, a batch job instance might create additional queues for internal purposes, for example, if a batch step contains an aggregator.